### DISCUSSION TITLE
Docker // Raspberry Pi

### DISCUSSION THREAD

**Discussion by ghost** [almarklein/timetagger#106] [Show and tell]
Made a docker image... [https://hub.docker.com/r/enj0ynet/timetagger](https://hub.docker.com/r/enj0ynet/timetagger)

I'm not the most diligent docker user let alone python-understander but it works on my machine - locked away on local network.

I may just end up using the actual service to feel more secure about data not getting lost.. and to support this great intuitive app, the best time tracker I've seen that is self-hostable.. great work Mr. AlmarKlein

**Comment by almarklein** [comment]
Thanks for sharing!
